analytics_data_api/v0/models.py

```diff
@@ -135,7 +135,7 @@ class Meta(BaseCourseModel.Meta):
 
     module_id = models.CharField(db_index=True, max_length=255)
     part_id = models.CharField(db_index=True, max_length=255)
-    correct = models.NullBooleanField()
+    correct = models.BooleanField(null=True)
```

Comment:
Run `make migrations` in the shell. It'll introduce new migrations due to this change.

Reply:
Doc link: https://docs.djangoproject.com/en/4.2/releases/4.0/#features-removed-in-4-0